### PR TITLE
koding: code quality fixes

### PR DESCRIPTION
**Pull request by rjeczalik**
I tried to rebase https://github.com/koding/koding/pull/9234 and I failed.

I'm resending https://github.com/koding/koding/compare/master...2a903ee62b7956cf2f211bc83573cc33086a9e48 it here again.

### CODE REVIEW

File: go/src/koding/klient/docker/docker_test.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func TestMain(m *testing.M) {
 	remote = d.NewClient(kiteURL.String())
 	err := remote.Dial()
 	if err != nil {
-		log.Fatal("err")
+		log.Fatalf("err")
```
Comment (ppknap) [inline go/src/koding/klient/docker/docker_test.go]: probably without quotes: `log.Fatalf(err)`

File: go/src/koding/klient/fs/fs_test.go [hunk 2]
```diff
@@ -67,7 +67,7 @@ func TestMain(m *testing.M) {
 	remote = remoteKite.NewClient(kiteURL.String())
 	err := remote.Dial()
 	if err != nil {
-		log.Fatal("err")
+		log.Fatalf("err")
```
Comment (ppknap) [inline go/src/koding/klient/fs/fs_test.go]: probably without quotes: `log.Fatalf(err)`

File: go/src/koding/klient/logfetcher/logfetcher_test.go [hunk 3]
```diff
@@ -38,15 +38,15 @@ func init() {
 	remote = remoteKite.NewClient("http://127.0.0.1:3639/kite")
 	err := remote.Dial()
 	if err != nil {
-		log.Fatal("err")
+		log.Fatalf("err")
```
Comment (ppknap) [inline go/src/koding/klient/logfetcher/logfetcher_test.go]: probably without quotes: `log.Fatalf(err)`

File: go/src/koding/klient/logfetcher/logfetcher_test.go [hunk 4]
```diff
@@ -38,15 +38,15 @@ func init() {
...
 	}
 
 	remoteKite2 := kite.New("remote2", "0.0.1")
 	remoteKite2.Config.Username = "remote2"
 	remote2 = remoteKite2.NewClient("http://127.0.0.1:3639/kite")
 	err = remote2.Dial()
 	if err != nil {
-		log.Fatal("err")
+		log.Fatalf("err")
```
Comment (ppknap) [inline go/src/koding/klient/logfetcher/logfetcher_test.go]: probably without quotes: `log.Fatalf(err)`

File: go/src/koding/klient/storage/storage_test.go [hunk 5]
```diff
@@ -58,7 +58,7 @@ func init() {
 	remote = remoteKite.NewClient("http://127.0.0.1:3637/kite")
 	err = remote.Dial()
 	if err != nil {
-		log.Fatal("err")
+		log.Fatalf("err")
```
Comment (ppknap) [inline go/src/koding/klient/storage/storage_test.go]: probably without quotes: `log.Fatalf(err)`

File: go/src/koding/klient/vagrant/handlers_test.go [hunk 6]
```diff
@@ -56,7 +56,7 @@ func TestMain(m *testing.M) {
 	remote = remoteKite.NewClient("http://127.0.0.1:3640/kite")
 	err = remote.Dial()
 	if err != nil {
-		log.Fatal("err")
+		log.Fatalf("err")
```
Comment (ppknap) [inline go/src/koding/klient/vagrant/handlers_test.go]: probably without quotes: `log.Fatalf(err)`

File: go/src/koding/workers/cmd/tunnelproxymanager/main.go [hunk 7]
```diff
@@ -21,7 +21,7 @@ const Name = "tunnelproxymanager"
 func main() {
 	c, err := configure()
 	if err != nil {
-		log.Fatal("Reading config failed: ", err.Error())
+		log.Fatalf("Reading config failed: ", err.Error())
```
Comment (ppknap) [inline go/src/koding/workers/cmd/tunnelproxymanager/main.go]: log.Fatalf("Reading config failed: #v", err)

**Review by ppknap — CHANGES_REQUESTED** [review]
Only small issues

**Comment by rjeczalik** [comment]
@cihangir 

pic

**Pull request closed without merging**